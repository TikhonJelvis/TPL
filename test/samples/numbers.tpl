print 0
print 1
print (1 + 2)
print (1 + 2 * 3)
print 100000000000000
print (10 * 10 * 10 * 10 * 10 * 10 * 10)
print (1 + 2 * 3 / 4 - 5)