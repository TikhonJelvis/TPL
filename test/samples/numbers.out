0
1
3
7
100000000000000
10000000
-4
null
